source/val/construct.h

```diff
@@ -109,6 +110,23 @@ class Construct {
   // calculated.
   ConstructBlockSet blocks(Function* function) const;
 
+  // Returns true if |dest| is structured exit from the construct. Structured
+  // exits depend on the construct type.
+  // Selection:
+  //  * branch to the associated merge
+  //  * branch to the merge or continue of the innermost loop containing the
+  //  selection
+  // Loop:
+  //  * branch to the associated merge or continue
+  // Continue:
+  //  * back-edge to the associated loop header
+  //  * branch to the associated loop merge
+  // Case:
```

Comment:
This lists case but the comment above says case never gets here?

Reply:
Yes, there is separate code to check cases. I'll update this comment.